src/radical/pilot/raptor/worker.py

```diff
@@ -195,73 +283,36 @@ def _call(self, data):
         args   = data.get('args',   [])
         kwargs = data.get('kwargs', {})
 
+        bak_stdout = sys.stdout
+        bak_stderr = sys.stderr
+
         try:
-            out = to_call(*args, **kwargs)
-            err = None
-            ret = 0
+            # redirect stdio to capture them during execution
+            sys.stdout = strout = StringIO()
+            sys.stderr = strerr = StringIO()
+
+            val = to_call(*args, **kwargs)
+            err = strout.getvalue()
+            ret = strerr.getvalue()
```

Comment:
Aren't these two lines the other way around?

Reply:
yes, part of the merge screwup - fixed already